.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Resolves #(issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
Why making it a checkbox list if we have to delete the unnecessary changes?
Might be simpler to just remove this sentence, and keep the checklist.